lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -145,9 +147,17 @@ internal sealed class REEFIMRUClient : IIMRUClient
                 .SetJobIdentifier(jobDefinition.JobName)
                 .Build();
 
-            _reefClient.SubmitAndGetDriverUrl(imruJobSubmission);
+            _httpEndPoint = _reefClient.SubmitAndGetDriverUrl(imruJobSubmission);
 
             return null;
         }
+
+        /// <summary>
+        /// DriverHttpEndPoint returned by IReefClient after job submission
+        /// </summary>
+        public IDriverHttpEndpoint DriverHttpEndpoint
```

Comment:
It will be a problem then. But note that this is a temporary unstable
interface in any case. SO let's go with this in any case since its going to
be changed soon.

On Mon, Oct 19, 2015 at 11:12 AM, Julia Wang <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs
> <https://github.com/apache/incubator-reef/pull/573#discussion_r42405847>:
>
> >
> >              return null;
> >          }
> > +
> > +        /// <summary>
> > +        /// DriverHttpEndPoint returned by IReefClient after job submission
> > +        /// </summary>
> > +        public IDriverHttpEndpoint DriverHttpEndpoint
>
> What if httpEndPoint hasn't got value but the user tries to access this
> property?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/573/files#r42405847>.
>